server.js

```diff
@@ -21,7 +24,67 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json({
+    message: "Welcome to the Netflix API!",
+    endpoints: listEndpoints(app),
+  });
+});
```

Comment:
when you list the endpoints you can add more info to document your API, for example you can filter the data by queries like title, you could add something like this:

```javascript
app.get("/", (req, res) => {
  const usageInfo = {
    description: "Get movies data.",
    queryParameters: {
      title: "Filter movies by title (case-insensitive).",
    },
  };

  const endpoints = listEndpoints(app);
  const moviesEndpointIndex = endpoints.findIndex(endpoint => endpoint.path === '/movies');

  if (moviesEndpointIndex !== -1) {
    endpoints[moviesEndpointIndex].usageInfo = usageInfo;
  }

  res.json({
    message: "Welcome to the Movies API!",
    endpoints: endpoints
  });
});
``` 
you can do the same for the year query ⭐